### PR TITLE
BACKLOG-13158 : update module for Jahia 8

### PR DESCRIPTION
**Pull request by Yousria**
## JIRA

https://jira.jahia.org/browse/BACKLOG-13158

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

**Review by tdraier — CHANGES_REQUESTED** [review]
use either require-capability in properties , or Require-Capability in instructions.
Actually , it's safer to use require-capability in properties, forget about changing instructions (and also put the signature there in jahia-module-signature )

File: pom.xml [hunk 1]
```diff
@@ -48,15 +48,16 @@
...
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>bootstrap3-modules</artifactId>
     <name>Bootstrap3 Modules</name>
     <packaging>pom</packaging>
-    <version>3.7.0-SNAPSHOT</version>
+    <version>4.0.0-SNAPSHOT</version>
     <properties>
         <jahia.package.version>1.0</jahia.package.version>
+        <jahia-module-signature>MCwCFFuYQJ8/rGH8aOS2Tw2tKOBmTGbhAhR07fDq3V5rsXzIfFS9FVHPzSyhng==</jahia-module-signature>
```
Comment (DameniMilo) [inline pom.xml]: this is not a module, it's just a package and it's never deployed on the platform, so I don't think you need it here.
To be confirmed by @tdraier

Comment (Yousria) [reply pom.xml]: I wasn't sure also

Comment (tdraier) [reply pom.xml]: yes, not needed in pom packaging, only for modules

**Review by DameniMilo — APPROVED** [review]

**Review by tdraier — APPROVED** [review]

**Pull request merged**